ykcompile/Cargo.toml

```diff
@@ -8,12 +8,12 @@ license = "Apache-2.0 OR MIT"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-yktrace = { path = "../yktrace" }
-ykpack = { path = "../ykpack" }
 dynasmrt = "1.0.0"
 hex = "0.4.2"
-libc = "0.2.80"
 lazy_static = "1.4.0"
+libc = "0.2.80"
+ykpack = { path = "../ykpack" }
+yktrace = { path = "../yktrace" }
```

Comment:
I personally like to put all related dependencies together with a newline separating them from the rest. In this case that would be all `yk*` dependencies.

Reply:
The beauty of this approach is that the `yk` dependencies live to next each other alphabetically. Or, put another way, it stops me bikeshedding on newlines :p